charger/evsewifi.go

```diff
@@ -172,6 +190,10 @@ func (wb *EVSEWifi) get(uri string) error {
 
 // Enable implements the api.Charger interface
 func (wb *EVSEWifi) Enable(enable bool) error {
+	ctrlAllowed, err := wb.controlAllowed()
+	if ctrlAllowed != true {
+	    return err
```

Comment:
Hier wird suggeriert, als würde die Box den Befehlen folgen, was sie aber absichtlich nicht tut. Dabei gibt es einen Fehler. Z.B. für ein Lastmanagementszenario eine Horrorsituation.

Reply:
Korrekt. Es könnte ein Fehler in err zurückgegeben werden. Besser wäre wenn ein Status in der GUI angezeigt werden würde, aber ich habe nichts entsprechendes in der API gesehen.
Alternativ kann enabled() in diesem Fall immer false rückgeben. Wäre das besser?
In jedem Fall wäre die Rückmeldung im Log, und nicht in der GUI.